rust-toolchain

```diff
@@ -0,0 +1 @@
+nightly
```

Comment:
Why depend on `nightly`? I'd prefer if we used `stable` for development.

Reply:
Dependencies didn't compile without it but I'll switch it to stable.